docs/quick-start/README.md

```diff
@@ -80,13 +80,15 @@ Clone your newly created repository and follow the instructions below.
    ```
    __Note:__ `config/dev.json` file is specific to your personal setup and should not be tracked in GIT.
 
-7. Bootstrap AWS CDK on target AWS account and region, e.g.:
+7. If you want to restrict access to your AWS User, attach the policy you find in `aws-policy.json` file.
```

Comment:
It's not quite clear where to attach this or how. Can you give or reference an example?